spec/nds_explore_spec.rb

```diff
@@ -1,37 +1,13 @@
 require 'spec_helper'
+require 'pp'
```

Comment:
Let's **not** include that here. It's good for **us** to include it to get things running, but part of the point of this lab is forthem to learn the require (first) and the `pp` second. To help them grok this, let's put a comment in the main file that basically reminds them to "be sure to include the pp library (SEE README for details!)".

This will catch "skimmers" and hopefully lower the barrier to succes for ppl who don't yet really understand what a library is.